steamer.go

```diff
@@ -156,11 +173,15 @@ func SearchHandler(w http.ResponseWriter, r *http.Request) {
 			Search  string
 			Breach  string
 			Sort    string
+			PageNum string
+			Limit string
```

Comment:
This formatting looks a little weird. I presume you haven't run gofmt on the code before making the PR. It's minor, but worth doing.

https://golang.org/cmd/gofmt/